package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "model-viewer",
-    "version": "5.3.3",
+    "version": "5.3.4",
```

Comment:
I would bump the package.json version separately from upgrading NPM dependencies, personally.

Reply:
yep, reverted